client/layout/index.jsx

```diff
@@ -110,7 +110,7 @@ module.exports = React.createClass( {
 					</Welcome>
 					<InviteMessage sites={ this.props.sites }/>
 					<EmailVerificationNotice user={ this.props.user } />
-					<NoticesList id="notices" notices={ notices.list } forcePinned={ 'post' === this.state.section } />
```

Comment:
I'm not really touching notices-list, just renaming, cleaning up the `is-pinned` attribute (since the new design will always be pinned).